docs/guides/web/nginx-mainline.md

```diff
@@ -0,0 +1,282 @@
+gpgcheck=1
+enabled=0
+gpgkey=https://nginx.org/keys/nginx_signing.key
+module_hotfixes=true
+```
+
+This code basically just lets you use the *Nginx*-made-and-hosted repositories for CentOS, and it allows you use the previously-mentioned “stable” branch if you want to. I mean, don’t. But you could.
+
+Save the file with Control-S (if using *nano*) and exit with Control-X.
+
+## Installing and Running Nginx
+
+Now, let’s enable the repository file you just made with one simple command:
+
+```bash
+sudo yum-config-manager --enable nginx-mainline
+```
+
+Then, install *Nginx*:
+
+```bash
+sudo dnf install nginx
+```
+
+The terminal will ask you if you’re fine with installing the repository’s GPG key. You need that, so choose “Y” for yes.
+
+Once the installation is done, start *Nginx* and enable it to automatically start on reboot all in one go with:
+
+```bash
+sudo systemctl enable --now nginx
+```
+
+To verify that the lastest version of *Nginx* has been installed, run:
+
+```bash
+nginx -v
+```
+
+From there, you could just start dropping HTML files into the `/usr/share/nginx/html/` directory to build a simple, static website. The configuration file for the default website/virtual host is called “default.conf” and it’s in `/etc/nginx/conf.d/`.
+
+## Configuring the Firewall
+
+!!! Note
+
+    If you are installing Nginx on a container such as LXD/LXC or Docker, you can just skip this part for now. The firewall should be handled by the host OS.
+
+If you try to view a web page at your machine’s IP address or domain name from another computer, you’re probably going to get a big fat nothing. Well, that’ll be the case as long as you have a firewall up and running.
+
+Here’s how to open up the necessary ports to actually see your web pages with *firewalld*, Rocky Linux’s default firewall with the `firewall-cmd` command. To add a new port, just run this:
+
+```bash
+sudo firewall-cmd --permanent --zone=public --add-port=80/tcp
+```
+
+Let’s break this down: 
+
+* The `-–permanent` flag tells the firewall to make sure this configuration is used every time the firewall is restarted, and when the server itself is restarted. 
+* `–-zone=public` tells the firewall to take incoming connections to this port from everyone.
+* Lastly, `–-add-port=80/tcp` tells the firewall to accept incoming connections over port 80, as long as they’re using the Transmission Control Protocol, which is what you want in this case.
+
+To repeat the process for SSL/HTTPS traffic, just run the command again, and change the number.
+
+```bash
+sudo firewall-cmd --permanent --zone=public --add-port=443/tcp
+```
+
+These configurations won’t take effect until you force the issue. To do that, tell *firewalld* to relead its configurations, like so:
+
+```bash
+sudo firewall-cmd --reload
+```
+
+Now, there’s a very small chance that this won’t work. In those rare cases, make *firewalld* do your bidding with the old turn-it-off-and-turn-it-on-again.
```

Comment:
No it should not fail...
Maybe this is a remarque, so we could add a Note admonition